docs/shared/react-tutorial/3-task-running.md

```diff
@@ -0,0 +1,179 @@
+- `options` - this is an object defining which executor options to use for the given target. Every Nx executor allows for options as a way to parameterize it's functionality.
+
+{% callout type="note" title="Target Configurations" %}
+In addition to allowing for option overrides at the command level, Nx also has a mechism called "configurations" for defining sets of option overrides. Common use-cases for this include a `development` and `production` configuration on your applications' `build` target.
+
+You can read more in [this recipe for Executor configurations](/recipe/use-executor-configurations).
+{% /callout %}
+
+These targets are sufficient for your `products` project, but if more targets were required in the future, you could create more targets by defining them here.
+
+{% callout type="note" title="Configuration-less Tasks"%}
+In addition to configuring your tasks via Nx executors in a `project.json` file, Nx also supports configuration-less setup, where Nx can run tasks as defined by the `scripts` in your project's `package.json` file.
+
+You can read more about configuring your project in [the Project Configuration reference page](/reference/project-configuration).
+{% /callout %}
+
+You can also check the `project.json` files of your other projects in the workspace (`admin`, `admin-e2e`, `store`, `store-e2e`, and `common-ui`) to see the targets already available for them.
+
+## Syntax for Running Tasks in Nx
+
+To run a target in Nx, use the following syntax:
+
+![Syntax for Running Tasks in Nx](/shared/react-tutorial/run-target-syntax.png)
+
+To see this in action, run the `test` target for our `products` project by running the command `npx nx test products`:
+
+```bash
+> npx nx test products
+
+> nx run products:test
+
+FAIL products libs/products/src/lib/products.spec.ts
+● Test suite failed to run
+
+    libs/products/src/lib/products.spec.ts:1:10 - error TS2724: '"./products"' has no exported member named 'products'. Did you mean 'Product'?
+
+    1 import { products } from './products';
+               ~~~~~~~~
+
+Test Suites: 1 failed, 1 total
+Tests: 0 total
+Snapshots: 0 total
+Time: 0.819 s
+Ran all test suites.
+
+———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+> NX Ran target test for project products (2s)
+
+    ✖    1/1 failed
+    ✔    0/1 succeeded [0 read from cache]
+
+```
+
+As we can see, our tests are currently failing as we haven't adjusted any of our tests since generating and making some changes to this project.
```

Comment:
You show the `jest.config.ts` file next and I expected to modify something in it to make the tests work.  I understand now that you were just showing that you can modify it, but this step ended without any sense of resolution for me.

I would prefer to fix the tests instead of showing the config file, if we only have space for one.

Reply:
I ended up pulling out the part about the `jest.config.ts` file, and I switched to use the `common-ui` library as an example here (its lint and test targets pass, making it so we don't call attention to a failing test, and then an unresulved conflict).